samples/algorithms/variational-algorithms/environment.yml

```diff
@@ -8,7 +8,7 @@ channels:
 dependencies:
   - notebook
   - matplotlib
-  - qsharp==0.21.2112.180703
+  - qsharp==0.21.2112182074b1
```

Comment:
Is this version number correct? I am just double checking, because it looks different from the others.

Reply:
Yes, but only while we keep this as a beta release.